src/MCPServer/lib/assets/workflow.json

```diff
@@ -1224,11 +1224,11 @@
             "exit_codes": {
                 "0": {
                     "job_status": "Completed successfully",
-                    "link_id": "75fb5d67-5efa-4232-b00b-d85236de0d3f"
+                    "link_id": "873b428f-2c86-42b6-b463-aeda925bf559"
```

Comment:
This looks good and is a sensible change. It's interesting to think about the SIP more completely now with the 1.9 changes for SFU, i.e. ensuring persistent identifiers arrive there does seem important!